src/app/page.tsx

```diff
@@ -28,6 +29,7 @@ export default async function Home({ searchParams }: HomeProps) {
 
 	return (
 		<>
+			<ScrollButton />
```

Comment:
[Suggestion] I'd suggest moving this to the bottom of the `MainArea`. That's where it is on the page. And things should be contained in sections such as `main`.